README.md

```diff
@@ -63,65 +66,97 @@ import project from '@dojo/web-editor/project';
 
 ### Editor
 
-This is a class which wraps `monaco-editor`, allowing the editor to seemlessly integrate with the project.  It will automatically
+This is a widget which wraps `monaco-editor`, allowing the editor to seemlessly integrate with the project.  It will automatically
 display files with the appropriate language intellisense as well as the file is edited, changes will automatically be sent to
 the project to keep it updated.
 
-The `Editor` constructor takes two arguments.  First, you need to provide a DOM element that will serve as the root for the editor.
-The second is an optional argument is a pass through of any options as specified in the `monaco.editor.IEditorOptions` interface.
+The `Editor` properties have two key properties:
 
-The class has only one method:
+* `filename` - This is the name of the file in the project which the editor should be displaying.
```

Comment:
Well, this one I debated, as like with widgets, you have to always set the properties.  So conditional future tense is not quite appropriate, but I get the point that it could be phrased better.